### ISSUE TITLE
Using different string-labels will casue "Key Error"

### ISSUE THREAD

**Issue by KeyaoZhao** [salesforce/LayoutDETR#3]
When I try Layout generation in the wild and use string-labels except "header", "body", "button", "disclaimer / footnote", it will cause "Key Error" in Line 346. It seems other string-labels in label_list are not define in json_temp and element_specs. So how can we try string-labels like "Logo" and "pre-header"? Thanks a lot!

**Comment by ningyu1991** [comment]
Yeah, so far this version of code only supports "header", "body", "button", "disclaimer / footnote" these four categories. We didn't find obvious difference if we assign "header" for what should be "logo" or "pre-header".

**Comment by KeyaoZhao** [comment]
> Yeah, so far this version of code only supports "header", "body", "button", "disclaimer / footnote" these four categories. We didn't find obvious difference if we assign "header" for what should be "logo" or "pre-header".
Thanks for your reply. I also wonder that "--strings='天猫狂欢季|2周年店庆|五重好礼送不停' --string-labels='header|body text|body text'"，model will write last body text content twice. Did I write it correctly? 
And I found the output is not as pleasing as your examples because the font is defined as 'Arial', did you modify the html file?

非常感谢您的回复！我也想请问下如果我尝试"--strings='天猫狂欢季|2周年店庆|五重好礼送不停' --string-labels='header|body text|body text'"，输出的结果中会写两次“五重好礼送不停”而没有写“2周年店庆”，请问我需要怎么修改嘛呢如果我需要多个body text的话？  
并且我发现我的输出效果没有您论文中的美观，generate.py中关于字体是只有Arial嘛，如果我也想生成同样
美观的图片是需要自己再次修改html文件还是代码可以直接完成呢？多谢~
